sql/core/src/main/scala/org/apache/spark/sql/sources/filters.scala

```diff
@@ -38,6 +38,14 @@ case class EqualTo(attribute: String, value: Any) extends Filter
 
 /**
  * A filter that evaluates to `true` iff the attribute evaluates to a value
+ * null safe and equal to `value`.
```

Comment:
Should I write like this?

A filter that evaluates to `true` if the attribute evaluates to a value equal to `value` and if both the attribute and the value are null.